labs/lab-06/tasks/mul-arrays/solution/mul_arrays.asm

```diff
@@ -14,34 +14,34 @@ extern printf
 global main
 
 main:
-    push ebp
-    mov ebp, esp
+    push rbp
+    mov rbp, rsp
 
-    xor ecx, ecx
+    xor rcx, rcx
 iterate_arrays:
-    xor eax, eax
-    xor ebx, ebx
-    xor edx, edx
-    mov al, byte[array1 + ecx]
-    mov bl, byte[array2 + ecx]
+    xor rax, rax
+    xor rbx, rbx
+    xor rdx, rdx
+    mov al, byte[array1 + rcx]
+    mov bl, byte[array2 + rcx]
```

Comment:
```suggestion
    mov al, [array1 + rcx]
    mov bl, [array2 + rcx]
```
Try to get rid of the unnecessary size specifiers wherever possible (when one of the operands is a register). This makes students wrongly believe it's always needed when it's not.